js/src/views/Web/web.js

```diff
@@ -67,28 +64,44 @@ export default class Web extends Component {
       );
     }
 
+    const { dappsUrl } = this.context.api;
+    const { url } = this.state;
+    if (!url || !token) {
+      return null;
+    }
+
+    const parsed = parseUrl(url);
+    let host = parsed.host;
+    let path = parsed.path;
+    if (!host) {
+      host = parsed.path.split(pathSep).slice(0, 1);
```

Comment:
Isn't this returning an array ? It's fine with the cast to String though, but still...